src/main/java/wolox/training/models/Book.java

```diff
@@ -94,63 +95,63 @@ public String getAuthor() {
   }
 
   public void setAuthor(String author) {
-    this.author = author;
+    this.author = Preconditions.checkNotNull(author, ErrorMessages.CANNOT_BE_NULL, "author");
```

Comment:
The precondition should be checked before assigning the value to the object :yum: